### PR TITLE
fix: TDE-402 GitHub actions git tag order

### PR DESCRIPTION
**Pull request by amfage**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM osgeo/gdal:ubuntu-small-latest
+FROM osgeo/gdal:ubuntu-small-3.5.0
```
Comment (paulfouquet) [inline Dockerfile:1]: I think this is not part of this PR?

**Review by paulfouquet — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (amfage) [reply Dockerfile:1]: Noo it is not.

Comment (paulfouquet) [reply Dockerfile:1]: It will get merged with the other one anyway :)
